waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/CloseableThriftHiveMetastoreIface.java

```diff
@@ -1,28 +0,0 @@
-/**
- * Copyright (C) 2016-2018 Expedia Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.hotels.bdp.waggledance.client;
-
-import java.io.Closeable;
-
-import org.apache.hadoop.hive.metastore.api.ThriftHiveMetastore.Iface;
-
-import com.hotels.hcommon.ssh.Tunnelable;
-
-public interface CloseableThriftHiveMetastoreIface extends Iface, Closeable, Tunnelable {
```

Comment:
I moved this to `hcommon-hive-metastore` so we can decouple implementations of the interface for pluggable components from `WaggleDanceCore`